backend/src/routes.ts

```diff
@@ -7,6 +11,141 @@ router.get('/', (_req, res) => {
   res.send('Hello World')
 })
 
-export {
-  router
-}
\ No newline at end of file
+// Player Routes
+router.get('/player', async (_req, res) => {
+  const players = await findMany<IPlayer>('players')
+  res.json(players)
+})
+
+router.get(`/player/:id`, async (req, res) => {
+  const { id } = req.params
+  const player = await findOne<IPlayer>('players', id)
+
+  res.json(player)
+})
+
+router.post('/player', async (req, res) => {
+  const data: IPlayer = req.body
+
+  if (!isPlayer(data)) {
+    res.status(500).end()
+    return
+  }
+
+  const player = await insert<IPlayer>('player', data)
```

Comment:
change `const player = await insert<IPlayer>('player', data)` to const player = await insert<IPlayer>('players', data)